keyboards/ducky/one2mini/rules.mk

```diff
@@ -0,0 +1,109 @@
+SRC += matrix.c
+
+# MCU name
+#MCU = at90usb1286
+MCU_FAMILY = NUMICRO
+MCU_SERIES = NUC123SD4AN0
+
+# linker script to use
+# it should exist either in <chibios>/os/common/ports/ARMCMx/compilers/GCC/ld/
+#  or <this_dir>/ld/
+MCU_LDSCRIPT = NUC123SD4AN0
+
+# startup code to use
+# is should exist in <chibios>/os/common/ports/ARMCMx/compilers/GCC/mk/
+MCU_STARTUP = NUC123SD4AN0
+
+# it should exist either in <chibios>/os/hal/boards/
+#  or <this_dir>/boards
+BOARD = NUC123SD4AN0
+
+# NUC123 series is Cortex M0
+MCU  = cortex-m0
+# ARM version, CORTEX-M0/M1 are 6, CORTEX-M3/M4/M7 are 7
+ARMV = 6
+
+# Processor frequency.
+#     This will define a symbol, F_CPU, in all source code files equal to the
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+F_CPU = 72000000
+
```

Comment:
Is this required?

Reply:
@elfmimi do you know the answer to this question?